File: Module 3/Mastering Swift 2_Code/B05115_13_codes/Projects/ObjectiveCProject/ObjectiveCProject/ViewController.m
//
//  ViewController.m
//  ObjectiveCProject
//
//  Created by Jon Hoffman on 2/11/15.
//  Copyright (c) 2015 Jon Hoffman. All rights reserved.
//

#import "ViewController.h"
#import "ObjectiveCProject-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

-(IBAction)messagePressed:(id)sender {
    MessageBuilder *mb = [[MessageBuilder alloc] init];
    
    self.messageView.text = [mb getPersonalizedMessage:self.nameField.text];
}

@end

File: Module 3/Mastering Swift 2_Code/B05115_13_codes/Projects/SwiftProject/SwiftProject/ViewController.swift
//
//  ViewController.swift
//  SwiftProject
//
//  Created by Jon Hoffman on 2/12/15.
//  Copyright (c) 2015 Jon Hoffman. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var messageBox: UITextView?
    @IBOutlet weak var nameField: UITextField?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    @IBAction func messagePressed(sender: UIButton) {
        let message = MessageBuilder()
        messageBox!.text = message.getPersonalizedMessageForName(nameField!.text);
    }


}

